lib/jsdom/living/nodes/HTMLElement-impl.js

```diff
@@ -75,6 +75,22 @@ class HTMLElementImpl extends ElementImpl {
     this._clickInProgress = false;
   }
 
+  get draggable() {
```

Comment:
Implement all the logic here. Test if `this._localName === "img"` or `this._localName === "a"`.